SparkPost/support-docs
# WE-285 Add Cookie Consent Banner

**What Changed**
- Adds the consent cookie banner, taken from the app

**How to verify**
- Run the site locally or visit the deploy preview
- The consent cookie banner should be visible
- Dismiss it
- The banner should not be visible after a hard refresh
- Open dev tools, open the Application tab. The cookie should be listed under `Cookies`.
